### PR TITLE
Minor Improvements to PatientRegistration; Drop External Results

### PR DESCRIPTION
**Pull request by gigincg**
Minor Improvements made to PatientRegistration UI
Remove Support for External Results

TODO: Remove All other usage of External Results

@ohcnetwork/care-fe-code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Comment by nihal467** [comment]
@gigincg 

image

- [ ] hide the create button in the external result tab, as we dont support patient creation using the external result

**Comment by gigincg** [comment]
@nihal467 Planning to remove the External Results Tab altogether. Any other references of External Results?

**Comment by nihal467** [comment]
> @nihal467 Planning to remove the External Results Tab altogether. Any other references of External Results?

nope, only this one reference remaining

**Comment by nihal467** [comment]
Based on the below conditions reviewing the PR :

- the existing redirection in the external result, even though the create button is visible, it is not crashing the page
- we are having a plan to remove the external tab entirely in the upcoming week
- external results are currently not in use at any production instances
- The existing patient registration flow is not affected.

**Comment by nihal467** [comment]
LGTM

File: src/Locale/en/Patient.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "personal_demographic": "Personal/Demographic Details",
+  "import_external_result": "Import From External Results",
+  "date_of_birth_note": "Date of Birth Note",
```
Comment (nihal467) [inline src/Locale/en/Patient.json:4]:
```suggestion
  "date_of_birth_note": "Please enter the correct date of birth for the patient",
```

File: src/Locale/en/Patient.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  "date_of_birth_note_description": "Please enter the correct date of birth for the patient",
```
Comment (nihal467) [inline src/Locale/en/Patient.json:5]:
```suggestion
  "date_of_birth_note_description": "Each patient in the system is uniquely identifiable by the number and date of birth. Adding incorrect date of birth can result in duplication of patient records.",
```

**Review by nihal467 — COMMENTED** [review]
Did the warning message about the date of birth note an intentional change? The modified note doesn't communicate why the user should enter the correct date of birth.

Note: the suggested changes are the existing warning message in the platform

**Comment by bodhish** [comment]
@gigincg any progress?

**Comment by gigincg** [comment]
@bodhish Closing it for now, doesn't seem salvageable. Will redo with Health Details

**Pull request closed without merging**